src/main/java/org/cactoos/text/PaddedStartText.java

```diff
@@ -57,21 +43,16 @@
      */
     public PaddedStartText(
         final Text text, final int length, final char symbol) {
-        this.origin = text;
-        this.symbol = symbol;
-        this.length = length;
-    }
-
-    @Override
-    public String asString() throws Exception {
-        final String original = this.origin.asString();
-        final int diff = this.length - original.length();
-        final StringBuilder builder = new StringBuilder();
-        for (int len = diff; len > 0; --len) {
-            builder.append(this.symbol);
-        }
-        builder.append(original);
-        return builder.toString();
+        super((Scalar<String>) () -> {
+            final String original = text.asString();
+            final int diff = length - original.length();
+            final StringBuilder builder = new StringBuilder();
+            for (int len = diff; len > 0; --len) {
+                builder.append(symbol);
+            }
+            builder.append(original);
+            return builder.toString();
+        });
```

Comment:
@Vatavuk WDYT about using `RepeatedText` to generate this pad string?

Reply:
@paulodamaso good idea! Refactored it in 2146d91